src/member.rs

```diff
@@ -41,19 +42,25 @@ pub(crate) enum UnitMessage<H: Hasher, D: Data, S: Signature> {
     RequestParents(NodeIndex, H::Hash),
     /// Response to a request for a full list of parents.
     ResponseParents(H::Hash, Vec<UncheckedSignedUnit<H, D, S>>),
+    /// Request by a node for the newest unit created by them
+    RequestNewest(NodeIndex),
+    ResponseNewest(Option<UncheckedSignedUnit<H, D, S>>),
 }
 
 impl<H: Hasher, D: Data, S: Signature> UnitMessage<H, D, S> {
     pub(crate) fn included_data(&self) -> Vec<D> {
         match self {
             Self::NewUnit(uu) => vec![uu.as_signable().data().clone()],
-            Self::RequestCoord(_, _) => Vec::new(),
             Self::ResponseCoord(uu) => vec![uu.as_signable().data().clone()],
-            Self::RequestParents(_, _) => Vec::new(),
             Self::ResponseParents(_, units) => units
                 .iter()
                 .map(|uu| uu.as_signable().data().clone())
                 .collect(),
+            Self::ResponseNewest(maybe_unit) => maybe_unit
```

Comment:
Yeah, all that makes sense.
For blocks availability, this might actually be a little concerning. Suppose we crash in session K, the period is P=600 (10 minutes) and we crashed at block 300 of this session (i.e., roughly block `P*K + 300` total) . Then when coming back, when starting session `K` again we will have block ~`P*K` imported, but people will start responding with our unit with block   `P*K + 300`. This means that we should be able to import 300 blocks in `5` seconds :P.

One ugly solution would be that in case we are responding to a Newest request and see that this node has  created >0 units, we first send him his round-0 unit and only then his most recent. At the receiving end on the other hand you increase the delay in case you receive any unit. 

Anything sounds bad here, but I'm afraid that the current solution might not even work "in practice" :P